Feedback

Chase-Hartley opened the pull request:
Code review request

Please check formatting, naming and logic

@Vanessa Calimag

Chase-Hartley committed:
Before working on practical

Chase-Hartley committed:
added allowance for lowercase letters in states.

Chase-Hartley committed:
Added loop to print all functions with a basic string formation.

Chase-Hartley committed:
Added dictionary and allow users to get the hex number for a certain colour.

Chase-Hartley committed:
Created program to compare a users string to a dictionary and keep count
of the occurance of each word.

lindsaymarkward commented:
Be careful with your mention. That should be @VanessaCalimag (the space breaks the name).

lindsaymarkward commented:
Please learn to make commit messages with the *imperative voice* -e.g. "Add" not "Added"

Chase-Hartley committed:
Merge branch 'master' of https://github.com/Chase-Hartley/CP1404 into feedback

Chase-Hartley committed:
add program that stores a users guitar collection.
add while loop to store the class objects to a list.
Check if guitar is vintage and print the list.
use formatting provided by prac.
